src/grow_worker/worker.py

```diff
@@ -0,0 +1,102 @@
+import jsonpickle
+import os
+import logging
+from typing import Optional
+from uuid import uuid4
+from pathlib import Path
+from celery import Celery
+from dataclasses import dataclass
+import subprocess
+
+from grow_runner.types import WorkFlowType
+
+LOGGER = logging.getLogger("grow_worker")
+
+app = Celery(
+    "omotes",
+    broker="amqp://user:bitnami@rabbitmq",
+    backend="rpc://user:bitnami@rabbitmq",
+    broker_connection_retry_on_startup=True,
+)
+
+
+@app.task(name="optimizer-task", bind=True)
+def optimize(self, job_id: uuid4, esdl_string: str):
+    LOGGER.info("optimizer worker started")
+    return jsonpickle.encode(
```

Comment:
Why are you actively adding jsonpickle here? Celery already serializes and deserializes to a format of our choosing: https://docs.celeryq.dev/en/stable/userguide/calling.html#serializers


Setting a new default: https://docs.celeryq.dev/en/stable/userguide/configuration.html#task-serializer

Reply:
I tried it that way: see commented out celery config. But unsuccessfully because we don't have the worker code in de orchestrator (or my local_test/test_app.py). Celery needs this to depickle. I tried with the worker code in the test_app, then it did work.